README.md

```diff
@@ -229,13 +319,23 @@ UIKit lets you align labels by top, bottom, or baseline. BonMot includes `BONTex
 
 You can use `BONTextAlignmentConstraint` programmatically or in Interface Builder. In code, use the convenience initializer:
 
+```swift
+BONTextAlignmentConstraint(item: someLabel,
+                      attribute: BONConstraintAttribute.CapHeight,
```

Comment:
No, that's a reasonable point. No harm done by leaving it a little wordier than necessary, and anyone who cares will likely know how to fix it.